grails-app/views/opus/_report.gsp

```diff
@@ -84,4 +87,84 @@
     </div>
 
     </script>
+
+    <script type="text/ng-template" id="mostRecentChange.html">
+    <div class="">
+        <label class="control-label">Show edited profiles from:</label>
+
+        <div class="btn-group" role="group" aria-label="List most recent changes with the following options.">
+            <button type="button" class="btn btn-default" ng-repeat="period in reportCtrl.periods"
+                    ng-click="reportCtrl.setPeriod(period)"
+                    ng-class="{active: reportCtrl.selectedPeriod.id == period.id}">{{ period.name }}</button>
+        </div>
+    </div>
+
+    <div class="well top-buffer" ng-show="reportCtrl.selectedPeriod.id == 'custom'">
+        <h4>Choose start and end dates</h4>
+        <div class="row">
+            <div class="col-md-6">
+                <label class="control-label col-sm-2" for="inputFromDate">Start date:</label>
+                <div>
+                    <p class="input-group">
+                        <input type="text" id="inputFromDate" class="form-control" ng-required="true"
```

Comment:
We'll need to update the styles of the date picker to make it look more like the example on the angular-bootstrap site (grey borders around buttons etc) and to change the appearance of the Done, Today and Clear buttons to be consistent with the rest of the site (I'd probably just make them look like links).

Will the datepicker work in all browsers? If it is limited to only html5 browsers, it might be better/easier to just use the html5 input type="date" field instead.